### PR TITLE
PMM-5777 Inventory add external exporter.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: inventory/agents_external_exporter_test.go [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+		res, err := client.Default.Agents.AddExternalExporter(&agents.AddExternalExporterParams{
+			Body: agents.AddExternalExporterBody{
+				ServiceID:    "",
+				RunsOnNodeID: genericNodeID,
+				ListenPort:   12345,
+			},
+			Context: pmmapitests.Context,
+		})
+		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "Empty Service ID.")
+		if !assert.Nil(t, res) {
```
Comment (percona-csalguero) [inline inventory/agents_external_exporter_test.go:185]: assert.NotNil

File: inventory/agents_external_exporter_test.go [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+		res, err := client.Default.Agents.AddExternalExporter(&agents.AddExternalExporterParams{
+			Body: agents.AddExternalExporterBody{
+				ServiceID:    serviceID,
+				RunsOnNodeID: genericNodeID,
+			},
+			Context: pmmapitests.Context,
+		})
+		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "invalid field ListenPort: value '0' must be greater than '0'")
+		if !assert.Nil(t, res) {
```
Comment (percona-csalguero) [inline inventory/agents_external_exporter_test.go:212]: NotNil

File: inventory/agents_external_exporter_test.go (hunk 1 above)
Comment (BupycHuk) [reply inventory/agents_external_exporter_test.go:185]: No, `nil` it's what I expect to have here and if it's not nil I should remove service manually.

**Comment by AlekSi** [comment]
Please ignore me there

File: inventory/agents_external_exporter_test.go [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+	"github.com/percona/pmm/api/inventorypb/json/client/services"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestExternalExporter(t *testing.T) {
+
```
Comment (AlekSi) [inline inventory/agents_external_exporter_test.go:17]: but I can't miss an empty line 😝

Comment (AlekSi) [reply inventory/agents_external_exporter_test.go:17]: (but please ignore me there)

**Pull request merged**